js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -637,3 +641,126 @@ impersonate=Impersonieren
+userModelAttribute=Benutzermodell-Attribut
+custom=Benutzerdefiniertes Attribut...
+ldapAttribute=LDAP-Attribut
+ldapAttributeHelp=Name des zugeordneten Attributs im LDAP-Objekt, zum Beispiel "cn", "sn", "mail", "street", usw.
+readOnly=Nur lesen
+readOnlyHelp=Ein schreibgeschütztes Attribut wird aus LDAP in das UserModel importiert, aber es wird nicht zurück in LDAP gespeichert, wenn der Benutzer in Keycloak aktualisiert wird.
+alwaysReadValueFromLdap=Wert immer aus LDAP lesen
+alwaysReadValueFromLdapHelp=Wenn aktiv, dann wird beim Lesen immer das LDAP-Attribut anstatt der Wert aus der Keycloak DB verwendet.
+isMandatoryInLdap=Ist in LDAP erforderlich
+isMandatoryInLdapHelp=Wenn aktiv, ist das Attribut in LDAP erforderlich. Wenn kein Wert in der Keycloak-DB vorhanden ist, wird ein leerer Wert gesetzt, und an LDAP weitergegeben.
+attributeDefaultValue=Attribut-StandardwertClick to apply
+attributeDefaultValueHelp=Wenn es keinen Wert in Keycloak DB gibt und das Attribut in LDAP erforderlich ist, dann wird dieser Wert an LDAP weitergegeben.
+isBinaryAttribute=Ist Binärattribut
+isBinaryAttributeHelp=Sollte für binäre LDAP-Attribute aktiviert werden.
+passwordPolicyHintsEnabled=Passwortrichtlinien-Hinweise aktiviert
+passwordPolicyHintsEnabledHelp=Gilt nur für beschreibbares MS Active Directory (MSAD). Wenn diese Option aktiviert ist, wird beim Aktualisieren des Passworts eines MSAD-Benutzers die LDAP_SERVER_POLICY_HINTS_OID-Erweiterung verwendet, was bedeutet, dass erweiterte MSAD-Passwortrichtlinien wie "Passworthistorie" oder "minimales Passwortalter" angewendet werden. Diese Erweiterung funktioniert nur für MSAD 2008 R2 oder neuer.
+managePriorities=Prioritäten verwalten
+alwaysReadEnabledValueFromLdap="Benutzer Aktiv" Status immer aus LDAP lesen
+alwaysReadEnabledValueFromLdapHelp=Wenn diese Option aktiviert ist, wird der Aktivierungs-/Deaktivierungsstatus des Benutzers immer aus MSAD gelesen, indem der entsprechende userAccountControl Wert überprüft wird.
+alwaysReadEnabledValueFromLdapLds="Benutzer Aktiv" Status immer aus LDAP lesen
+alwaysReadEnabledValueFromLdapLdsHelp=Wenn eingeschaltet, wird der Aktivierungs-/Deaktivierungsstatus des Benutzers immer aus der MSAD-LDS gelesen, indem der Wert msDS-UserAccountDisabled überprüft wird.
+ldapDerFormatted=DER Formatiert
+ldapDerFormattedHelp=Aktivieren Sie diese Option, wenn das Zertifikat in LDAP DER-formatiert ist und nicht PEM-formatiert.
+kerberosRealm=Kerberos-Realm
+kerberosRealmHelp=Name des Kerberos-Realms, zum Beispiel: FOO.ORG
+serverPrincipal=Server-Prinzipal
+serverPrincipalHelp=Vollständiger Name des Server-Principals für den HTTP-Dienst einschließlich Server- und Domänenname. Zum Beispiel: HTTP/host.foo.org@FOO.ORG
+keyTab=Key Tab
+keyTabHelp=Speicherort der Kerberos KeyTab-Datei, die die Anmeldedaten des Server-Prinzipals enthält, zum Beispiel: /etc/krb5.keytab.
```

Comment:
KeyTab or Key Tab (see before)?

Reply:
I now use: `Key-Tab` and `Key-Tab-Datei`